src/amo/components/AddonReviewList.js

```diff
@@ -107,6 +107,15 @@ export class AddonReviewListBase extends React.Component {
     return `${this.addonURL()}reviews/`;
   }
 
+  setReviewBody(review) {
+    let reviewBody = <p><LoadingText /></p>;
+    if (review) {
+      const content = sanitizeHTML(nl2br(review.body), ['br']);
+      reviewBody = <p dangerouslySetInnerHTML={content} />;
```

Comment:
I'd rename `content` to something more meaningful; `content` is vague but `reviewBodySanitized` would be good.